src/backend/common/queries/insight_query.py

```diff
@@ -0,0 +1,42 @@
+from typing import Any, Generator, List
+
+from backend.common.models.insight import Insight
+from backend.common.models.keys import Year
+from backend.common.queries.database_query import CachedDatabaseQuery
+from backend.common.queries.dict_converters.insight_converter import (
+    InsightConverter,
+    InsightDict,
+)
+from backend.common.tasklets import typed_tasklet
+
+
+class InsightsByNameQuery(CachedDatabaseQuery[List[Insight], List[InsightDict]]):
```

Comment:
Is this query used anywhere?

Reply:
This is currently not used but I had it in there for maybe future use if we wanted to add endpoints for specific insights. But not super sure that's actually important right now. I'll remove and we can always put back in later.